src/Servers/HttpSys/src/Resources.resx

```diff
@@ -117,6 +117,11 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="Exception_AccessDenied" xml:space="preserve">
+    <value>The prefix '{0}' is not registered. Please run the following command as Administrator to register this prefix:
+netsh http add urlacl url={0} user={1}
+See "Preregister URL prefixes on the server" on https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/httpsys#configure-windows-server for more information.</value>
```

Comment:
```suggestion
See "Preregister URL prefixes on the server" on https://go.microsoft.com/fwlink/?linkid=2127065 for more information.</value>
```

Reply:
Done, thanks for creating the link @shirhatti.